administration/src/mui-modules/NonIdealState.tsx

```diff
@@ -0,0 +1,20 @@
+import { Container, Stack, Typography, useTheme } from '@mui/material'
+import React, { ReactElement } from 'react'
+
+/* eslint-disable react/destructuring-assignment */
```

Comment:
You can deconstruct it without adding an eslint rule:
`const NonIdealState = ({ title, icon, description }:{ title: string; icon: ReactElement; description: string }`